### ISSUE TITLE
Cannot read properties of undefined (reading 'ignore') - Error report for exile-diary 1.1.0

### ISSUE THREAD

**Issue by Didrikos** [Qt-dev/exile-diary#190]
Error:
```
TypeError: Cannot read properties of undefined (reading 'ignore')
    at getVendorRecipeValue (C:\Program Files\Exile Diary Reborn\resources\app.asar\build\main\modules\ItemPricer.js:402:32)
    at Object.<anonymous> (C:\Program Files\Exile Diary Reborn\resources\app.asar\build\main\modules\ItemPricer.js:611:51)
    at step (C:\Program Files\Exile Diary Reborn\resources\app.asar\build\main\modules\ItemPricer.js:33:23)
    at Object.next (C:\Program Files\Exile Diary Reborn\resources\app.asar\build\main\modules\ItemPricer.js:14:53)
    at fulfilled (C:\Program Files\Exile Diary Reborn\resources\app.asar\build\main\modules\ItemPricer.js:5:58)
```
OS: Windows NT 10.0.19045
App: exile-diary 1.1.0

Error happens while trying to generate the map log which I assume contains 6s items, only getting data with maps without them.

**Comment by Qt-dev** [comment]
I believe you can fix it by going to the Item Filter tab in the settings, and changing any of these (Just untick a box and tick it again).
I am pushing a new version soon with a fix for this anyway

**Comment by Qt-dev** [comment]
This should be fixed with 1.1.1, feel free to reopen the issue if it's not
